app/src/main/java/io/apicurio/registry/storage/impl/sql/SQLServerSqlStatements.java

```diff
@@ -44,51 +44,81 @@ public String isDatabaseInitialized() {
      */
     @Override
     public String getNextSequenceValue() {
-        return String.join(" ", "MERGE INTO sequences AS target", "USING (VALUES  (?)) AS source (seqName)",
-                "ON (target.seqName = source.seqName)", "WHEN MATCHED THEN",
-                "UPDATE SET seqValue = target.seqValue + 1", "WHEN NOT MATCHED THEN",
-                "INSERT (seqName, seqValue)", "VALUES (source.seqName, 1)", "OUTPUT INSERTED.seqValue;");
+        return """
+                MERGE INTO sequences AS target
+                USING (VALUES  (?)) AS source (seqName)
+                ON (target.seqName = source.seqName)
+                WHEN MATCHED THEN
+                UPDATE SET seqValue = target.seqValue + 1
+                WHEN NOT MATCHED THEN
+                INSERT (seqName, seqValue)
+                VALUES (source.seqName, 1)
+                OUTPUT INSERTED.seqValue
+                """;
     }
 
     /**
      * @see io.apicurio.registry.storage.impl.sql.SqlStatements#resetSequenceValue()
      */
     @Override
     public String resetSequenceValue() {
-        return String.join(" ", "MERGE INTO sequences AS target",
-                "USING (VALUES (?, ?)) AS source (seqName, seqValue)", "ON (target.seqName = source.seqName)",
-                "WHEN MATCHED THEN", "UPDATE SET seqValue = ?", "WHEN NOT MATCHED THEN",
-                "INSERT (seqName, seqValue)", "VALUES (source.seqName, source.seqValue)",
-                "OUTPUT INSERTED.seqValue;");
+        return """
+                MERGE INTO sequences AS target
+                USING (VALUES (?, ?)) AS source (seqName, seqValue)
+                ON (target.seqName = source.seqName)
+                WHEN MATCHED THEN
+                UPDATE SET seqValue = ?
+                WHEN NOT MATCHED THEN
+                INSERT (seqName, seqValue)
+                VALUES (source.seqName, source.seqValue)
+                OUTPUT INSERTED.seqValue
+                """;
     }
 
     /**
      * @see SqlStatements#upsertContentReference()
      */
     @Override
     public String upsertContentReference() {
-        return String.join(" ", "MERGE INTO content_references AS target",
-                "USING (VALUES (?, ?, ?, ?, ?)) AS source (contentId, groupId, artifactId, version, name)",
-                "ON (target.contentId = source.contentId AND target.name = source.name)",
-                "WHEN NOT MATCHED THEN", "INSERT (contentId, groupId, artifactId, version, name)",
-                "VALUES (source.contentId, source.groupId, source.artifactId, source.version, source.name);");
+        return """
+                MERGE INTO content_references AS target
+                USING (VALUES (?, ?, ?, ?, ?)) AS source (contentId, groupId, artifactId, version, name)
+                ON (target.contentId = source.contentId AND target.name = source.name)
+                WHEN NOT MATCHED THEN", "INSERT (contentId, groupId, artifactId, version, name)
```

Comment:
The quotes `THEN", "INSERT` do not seem right.

Reply:
Good catch!